src/lib.rs

```diff
@@ -84,4 +132,38 @@ pub fn init<R: Runtime>() -> TauriPlugin<R> {
             Ok(())
         })
         .build()
-}
\ No newline at end of file
+}
+
+#[cfg(target_os = "linux")]
+pub fn init<R: Runtime>() -> TauriPlugin<R> {
+    use async_std::task;
+
+    let (tx, rx) = mpsc::channel::<os::GtkThreadCommand>();
+    let rx = Arc::new(Mutex::new(rx));
+
+    let rx = rx.clone();
+    glib::timeout_add_local(Duration::from_millis(200), move || {
+        let rx = rx.lock().unwrap();
+        if let Ok(cmd) = rx.try_recv() {
+            match cmd {
+                os::GtkThreadCommand::ShowContextMenu { pos, items, window } => {
+                    task::block_on(async {
```

Comment:
Can [async_runtime::block_on](https://docs.rs/tauri/latest/tauri/async_runtime/fn.block_on.html) be used here? This way we don't have to have two async runtimes.

Reply:
I am going to apply this in a new PR.
Thanks a lot!